### PR TITLE
db schema

### PR DESCRIPTION
**Pull request by josuevalo**

### CODE REVIEW

**Review by julia-gatina — APPROVED** [review]
Looks awesome!

**Review by lucyshen7 — APPROVED** [review]

File: express-back-end/db/schema/03_species.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS species CASCADE;
+CREATE TABLE species ( 
+  id SERIAL PRIMARY KEY NOT NULL,
+  common_name VARCHAR(255) NOT NULL,
+  category VARCHAR(255) NOT NULL,
+  description TEXT NOT NULL,
+  photo TEXT NOT NULL, 
+  light_level VARCHAR(255) NOT NULL,
+  soil_type VARCHAR(255) NOT NULL,
+  toxix BOOLEAN NOT NULL,
```
Comment (lucyshen7) [inline express-back-end/db/schema/03_species.sql:10]: should be toxic 😃

Comment (lucyshen7) [reply express-back-end/db/schema/03_species.sql:10]: do we also want to add scientific_name?

File: express-back-end/db/schema/08_followers.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+DROP TABLE IF EXISTS followers CASCADE;
+CREATE TABLE followers ( 
+  id SERIAL PRIMARY KEY NOT NULL,
+  user_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
```
Comment (lucyshen7) [inline express-back-end/db/schema/08_followers.sql:4]: this is just missing follower_id INTEGER REFERENCES users(id) ON DELETE CASCADE

File: express-back-end/db/schema/04_user_plants.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS user_plants CASCADE;
+CREATE TABLE user_plants (
```
Comment (lucyshen7) [inline express-back-end/db/schema/04_user_plants.sql:2]: should we make this table 'users_plants' instead?

File: express-back-end/db/schema/01_users.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS users CASCADE;
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(255) NOT NULL,
+  email VARCHAR(255) NOT NULL,
+  password VARCHAR(255) NOT NULL,
+  avatar_url VARCHAR(255) NOT NULL
```
Comment (lucyshen7) [inline express-back-end/db/schema/01_users.sql:7]: should avatar be TEXT NOT NULL? (similar to photo in posts)

File: express-back-end/db/schema/01_users.sql [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+  quote VARCHAR(255) NOT NULL,
```
Comment (lucyshen7) [inline express-back-end/db/schema/01_users.sql:8]: quote should probably be TEXT NOT NULL

**Review by lucyshen7 — COMMENTED** [review]
please see comments in code, thanks!

**Pull request merged**